app.json

```diff
@@ -20,8 +20,7 @@
     },
     "android": {
       "package": "org.timeoverflow.mobileapp",
-      "versionCode": 5,
-      "permissions": []
+      "versionCode": 8
```

Comment:
Is this the supported Android version? or the version of our app?

Reply:
The version of our app, every time that we upload apk to play store shoud be with diferent versionCode